src/modules/utils.ts

```diff
@@ -152,3 +152,30 @@ export function stringifyCommand(
 		(options.length ? ["", ...options].join("\n-\t") : "")
 	);
 }
+
+function stringifyContextMenuCommand(
+	interaction: Discord.ContextMenuCommandInteraction
+): string {
+	let target;
+	if (interaction.isUserContextMenuCommand()) {
+		target = `${interaction.targetUser}`;
+	} else if (interaction.isMessageContextMenuCommand()) {
+		target = `https://discord.com/channels/${
+			interaction.guildId ?? "@me"
+		}/${interaction.channelId}/${interaction.targetId}`;
+	}
+
+	return `[${interaction.user.tag}]: \`${interaction.commandName}\` @ ${target}`;
+}
+
+export function stringifyCommand(
+	interaction: Discord.CommandInteraction
+): string {
+	if (interaction.isChatInputCommand()) {
+		return stringifyChatInputCommand(interaction);
+	} else if (interaction.isContextMenuCommand()) {
+		return stringifyContextMenuCommand(interaction);
+	} else {
+		throw new Error("Unknown command type");
```

Comment:
Add a `logger.error({ interaction }, "Failed to stringify command due to unknown type)` here